build-support/bin/generate_travis_yml.py

```diff
@@ -114,22 +115,23 @@ def all_entries(cls) -> List[Dict[str, str]]:
 class PythonVersion(Enum):
     py37 = "py37"
     py38 = "py38"
+    py39 = "py39"
 
     def __str__(self) -> str:
         return str(self.value)
 
     @property
     def number(self) -> int:
-        return {self.py37: 37, self.py38: 38}[self]  # type: ignore[index]
+        return {self.py37: 37, self.py38: 38, self.py39: 39}[self]  # type: ignore[index]
 
     @property
     def decimal(self) -> float:
-        return {self.py37: 3.7, self.py38: 3.8}[self]  # type: ignore[index]
+        return {self.py37: 3.7, self.py38: 3.8, self.py39: 3.9}[self]  # type: ignore[index]
 
     def default_stage(self, *, is_bootstrap: bool = False) -> Stage:
         if is_bootstrap:
-            return {self.py37: Stage.bootstrap, self.py38: Stage.bootstrap_cron}[self]  # type: ignore[index]
-        return {self.py37: Stage.test, self.py38: Stage.test_cron}[self]  # type: ignore[index]
+            return {self.py37: Stage.bootstrap, self.py38: Stage.bootstrap_cron, self.py39: Stage.bootstrap_cron}[self]  # type: ignore[index]
+        return {self.py37: Stage.test, self.py38: Stage.test_cron, self.py39: Stage.test_cron}[self]  # type: ignore[index]
```

Comment:
When testing, you will want to invert these so that PY39 runs during daily and the others during nightly. Although, last time I checked Py39 shards are already working and the only thing left is the Build Wheels shards, which should happen no matter what during daily.

Reply:
I'm not super-familiar with our travis configuration setup. Are you saying that this line should be:
`        return {self.py37: Stage.test_cron, self.py38: Stage.test_cron, self.py39: Stage.test}[self]  # type: ignore[index]`

with the effect of running Python 3.9 by default in ordinary builds, and running Python 3.8/Python 3.7 during nightly "cron" builds?

Reply:
Exactly, but only for the sake of testing. Before merging, we want to revert it back to what you have currently